docs/conf/portland/2023/writing-day-cheatsheet.rst

```diff
@@ -0,0 +1,7 @@
+:template: {{year}}/generic.html
+:banner: _static/conf/images/headers/writing-day.png
+
+Writing Day Cheat Sheet
+=======================
+
+{% include "conf/virtual/writing-day-cheatsheet.rst" %}
```

Comment:
Do we still maintain a non `virtual` writing day and Welcome wagon?

Reply:
I need to check